plugin.js

```diff
@@ -0,0 +1,50 @@
+const AdminBro = require('admin-bro')
+
+module.exports = {
+  name: 'AdminBroExpressjs',
+  version: '0.1.0',
+
+  register: async (app, options) => {
```

Comment:
I would rename method to `expressRouter` or `buildExpressRouter` - `register` is a way how to handle plugins in hapijs.